src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -46,50 +44,15 @@ export class OverviewRoute extends React.Component<OverviewRouteProps, State> {
     // no-op
   }
 
-  componentDidMount() {
-    if (this.shouldShowCollectionRailTest) {
-      this.trackingCollectionsRailTest()
-    }
-  }
-
-  @track(() => {
-    // TODO: remove after CollectionsRail a/b test
-    const experiment = "artist_collections_rail"
-    const variation = sd.ARTIST_COLLECTIONS_RAIL
-
-    return {
-      action_type: Schema.ActionType.ExperimentViewed,
-      experiment_id: experiment,
-      experiment_name: experiment,
-      variation_id: variation,
-      variation_name: variation,
-      nonInteraction: 1,
-    }
-  })
-  trackingCollectionsRailTest() {
-    // no-op
-  }
-
   maybeShowGenes() {
     const { isReadMoreExpanded } = this.state
     const hasNoBio = !this.props.artist.biography_blurb.text
 
     return isReadMoreExpanded || hasNoBio
   }
 
-  get shouldShowCollectionRailTest() {
```

Comment:
This removes the limit that scopes the rails for just the ones included in the static list.